### PR TITLE
[[ Bug 20173 ]] Build universal solution for LCFM

### PR DESCRIPTION
**Pull request by montegoulding**

### CODE REVIEW

**Comment by livecodeali** [comment]
@livecode-vulcan review ok bae0567

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok bae05676e7faad6772e8200a5b93aac344640fbb

File: buildbot.mk [hunk 1]
```diff
@@ -286,22 +286,25 @@ distmac-extract:
...
 	$(buildtool_command) --platform win-x86 --stage fmpackage --debug; \
 	$(buildtool_command) --platform win-x86_64 --stage fmpackage --debug; \
+	$(buildtool_command) --platform universal --stage fmpackage --debug; \
 	find . -maxdepth 1 -name 'LiveCodeForFM-Mac-*.fmp12' -print0 \
 	    | xargs -0 zip -r LiveCodeForFM-Mac-Solution.zip; \
 	find . -maxdepth 1 -name 'LiveCodeForFM-Win-x86-*.fmp12' -print0 \
 	    | xargs -0 zip -r LiveCodeForFM-Win-x86-Solution.zip; \
 	find . -maxdepth 1 -name 'LiveCodeForFM-Win-x86_64-*.fmp12' -print0 \
 	    | xargs -0 zip -r LiveCodeForFM-Win-x86_64-Solution.zip
+	find . -maxdepth 1 -name 'LiveCodeForFM-[1-9]*.fmp12' -print0 \
```
Comment (montegoulding) [inline buildbot.mk:296]: @livecodeali this file doesn't appear to have been uploaded... any clues?

**Comment by livecode-vulcan** [comment]
😎 test success bae05676e7faad6772e8200a5b93aac344640fbb

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/534)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/549)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/523)
* try-community-universal-ios-iphoneos10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos10.3/builds/466)
* try-community-universal-ios-iphonesimulator10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator10.3/builds/469)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/529)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/537)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/532)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/549)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/532)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/542)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/564)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/528)

**Pull request merged**